types/mapbox__mapbox-sdk/index.d.ts

```diff
@@ -1699,7 +1700,7 @@ declare module '@mapbox/mapbox-sdk/services/uploads' {
          * Create an upload.
          * @param config
          */
-        createUpload(config: { mapId: string; url: string; tilesetName?: string }): MapiRequest;
+        createUpload(config: { mapId: string; url: string; tilesetName?: string, name?: string, private?: boolean }): MapiRequest;
```

Comment:
I'm not sure that I see a `private` parameter on the config object passed into `createUpload` here, is that intentional? Or is this coming in a new version of the mapbox-sdk?

https://github.com/mapbox/mapbox-sdk-js/blob/fc74a74f6c48d7373aec06548379f0c534f4d298/services/uploads.js#L121

Reply:
Looks like the SDK needs an update as well. [Here](https://docs.mapbox.com/api/maps/#create-an-upload) are the official docs from Mapbox where they support this. I'll also create a PR for the sdk to include the parameter

Reply:
Once https://github.com/mapbox/mapbox-sdk-js/pull/373 gets merged I'll update the code here and the Mapbox sdk version